lib/services/user/user_service.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:survey/models/user_info.dart';
+import 'package:survey/services/api/api_service.dart';
+import 'package:survey/services/http/http_service.dart';
+import 'package:survey/services/locator/locator_service.dart';
+
+abstract class UserService {
+  static const meEndPoint = "/me";
```

Comment:
I'd recommend to create another abs class where `Endpoint.profile = '/me' `.
The different is that you can group all of your endpoint to 1 place (up to you to manage the other Service too or you want to create 1 per Service). The advantages are:
- If we expand the UserService with more functions, it won't become a giant class.
- Clearer visibility
- Reduce duplication in the naming i.e remove `EndPoint` as a suffix. Btw it's `endpoint` so in English we should write as Endpoint instead of EndPoint.

Reply:
@sleepylee 
The main point I want to keep those constants and other things relating to this service in the same place so that this service can stand alone.
And, when we want to remove or copy this service, we can do it easier.